src/app/(auth)/styles.tsx

```diff
@@ -39,6 +42,19 @@ export const PCentered = styled(P)`
   text-align: center;
 `;
 
+export const AuthHeading = styled.h1`
```

Comment:
could we use the `H1` from the global styles? I think its `font-weight` should be `600`, so you can just go ahead and update it.

Reply:
updated!!